dev-util/hyprwayland-scanner/hyprwayland-scanner-0.3.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A Hyprland implementation of wayland-scanner, in and for C++"
+
+HOMEPAGE="https://github.com/hyprwm/hyprwayland-scanner/"
+
+inherit cmake
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/hyprwm/hyprwayland-scanner.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/hyprwm/hyprwayland-scanner/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
I'd suggest to use ${P}.gh.tar.gz, as it's a tag release from github.. just as a convention..